File: TYPESCRIPT PROGRAMMING/Test66.ts
// 66. PALINDROME NUMBER
const Palindrome = (num: number): void => {
    let NUM: number = num, rev: number = 0;
    while(num != 0) {
        let rem: number = num % 10;
        rev = (rev * 10) + rem;
        num = Math.floor(num / 10);
    }
    let res: string = (NUM == rev) ? 
        `${NUM} Is A Palindrome Number.` : `${NUM} Is Not A Palindrome Number.`;
    console.log(res);
}
Palindrome(121);
Palindrome(153);

File: JAVA PROGRAMMING/Test66.java
// 66. PALINDROME NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Num Value: ");
        int num = in.nextInt();

        int NUM = num, rev = 0;
        while(num != 0) {
            int rem = num % 10;
            rev = (rev * 10) + rem;
            num /= 10;
        }
        String res = (NUM == rev) 
            ? NUM+" is a palindrome number." : NUM+" is not a palindrome number.";
        System.out.println(res);
        in.close();
    }
}